CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # IdentityCache changelog
 
-#### Unreleased
+#### 0.3.2
 
+- Fetch association returns relation or array depending on the configuration. It was only returning a relation for cache_has_many fetch associtions methods.
```

Comment:
typo: `associtions` -> `association`

Also, I think we should start referencing pull requests or issues in changelog entries by adding `(#276)` to the end to entry